accounts/forms.py

```diff
@@ -471,6 +471,11 @@ class StudyLocation(FormChoiceEnum):
         lookit = ("1", _("here on the Lookit platform"))
         external = ("2", _("on other websites"))
 
+    class StudyLanguage(FormChoiceEnum):
+        all = ("*", _("All languages"))
+        english = ("en", _("English"))
+        japanese = ("ja", _("Japanese"))
+
```

Comment:
Wow!  This is great.  The filter drop-down would need to have the languages of all available studies.

Reply:
How to do that, I am unsure at this moment.

Reply:
At present, we believe there are only ja and en studies. But did you mean you'd like to automatically populate this list by querying the available studies?

Reply:
Correct. If someone were to create a Spanish study, they wouldn't be able to use this filter.

Reply:
@okaycj is correct, if we want to create a language filter for studies (which is a great idea!), we should go ahead and build the capacity to do it for any of the other languages we support.